src/vm/create.spec.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-env jest */
+
+import eventToPromise from 'event-to-promise'
+import {
+  map,
+  find
+} from 'lodash'
+
+import {
+  config,
+  getNetworkId,
+  getSrId,
+  waitObjectState,
+  xo
+} from './../util'
+
+// ===================================================================
+
+describe('.create()', () => {
+  let vmId
+  let networkId
+  let srId
+  let serverId
+  let vmsId = []
+
+  // ----------------------------------------------------------------------
+
+  function createVmTest () {
+    return xo.call('vm.create', {
+      name_label: 'vmTest',
+      template: getTemplateId(config.templates.debian),
+      VIFs: []
+    })
+  }
+
+  function deleteVm (id) {
+    return xo.call('vm.delete', {id, delete_disks: true})
+  }
+
+  function getTemplateId (nameTemplate) {
+    return find(xo.objects.all, {type: 'VM-template', name_label: nameTemplate}).id
+  }
+
+  // ----------------------------------------------------------------------
+
+  beforeAll(async () => {
+    serverId = await xo.call('server.add', config.masterServer)
+    await eventToPromise(xo.objects, 'finish')
+
+    networkId = getNetworkId()
+    srId = getSrId()
+  })
+
+  // ----------------------------------------------------------------------
+
+  afterAll(async () => {
+    await Promise.all(map(
+      vmsId,
+      vmId => deleteVm(vmId)))
```

Comment:
You can replace `vmId => deleteVm(vmId)` with `deleteVm` (maybe not relevant, see next comment).